Makefile

```diff
@@ -15,6 +15,7 @@ teardown:
 
 .PHONY: build
 build: build-buf build-js
+	npm run copy-dts
```

Comment:
I would add this to the `build-js` target (or add it to the npm `build` script, as in my other comment)